cEP-0019.md

```diff
@@ -0,0 +1,208 @@
+Meta-review System
+===========================
+
+|Metadata|                                   |
+|--------|-----------------------------------|
+|cEP     |0019                               |
+|Version |1.0                                |
+|Title   |Meta-review System                 |
+|Authors |Boxuan Li <liboxuan@connect.hku.hk>|
+|Status  |Proposed                           |
+|Type    |Process                            |
+
+Abstract
+--------
+
+This cEP describes the details of the process of implementing the meta-review
+system as a part of the
+[GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).
+
+Background
+----------
+
+People including the author of pull request respond to comments by attaching
+emojis. Those emojis are called meta-review, or
```

Comment:
> Those emojis are called meta-review

This isnt correct.

We are going to use reactions as a mechanism to build a meta-review system.